claat/proto-renderer/html/templates-tests/button_test.go

```diff
@@ -0,0 +1,60 @@
+package htmltests
+
+import (
+	"go/build"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	"github.com/googlecodelabs/tools/claat/proto-renderer/html"
+	"github.com/googlecodelabs/tools/claat/proto-renderer/testing-utils"
+	"github.com/googlecodelabs/tools/third_party"
+)
+
+func TestRenderButtonTemplate(t *testing.T) {
+	// TODO: Generalize file reading logic
+	var fOuts []string
+	files := []string{
+		"plain.txt",
+		"download.txt",
+	}
+	for _, f := range files {
+		linkFileRelDir := "src/github.com/googlecodelabs/tools/claat/proto-renderer/html/templates-tests/testdata/Button"
```

Comment:
I don't know how to feel about constants once I make a helper for the file-based tests, since is better for file names to be collocated with their tests. Some part of it will be made into constants (`src/github.com/googlecodelabs/tools/claat/proto-renderer` and `templates-tests/testdata`), but not the file names. I'm working on this in another PR/Branch

These will be declared at package level on leaf-tests, yet referred to again in the "identity" tests of their `oneof` grouping types (for `InlineContent` at least now).

My question: Is it worth making the aforementioned "identity" tests?

Reply:
#298